apps/backoffice-v2/public/locales/en/translation.json

```diff
@@ -1 +1,3 @@
-{}
+{
+    "home.greeting": "Welcome"
+}
```

Comment:
```suggestion
{
    "home": {
      "greeting": "Welcome"
    }
}
```

Reply:
Was about to change it, just added the whole by mistake.